src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/HtmlFieldMonacoEditorSettingsDriver.cs

```diff
@@ -44,16 +45,27 @@ public override async Task<IDisplayResult> UpdateAsync(ContentPartFieldDefinitio
 
                 await context.Updater.TryUpdateModelAsync(model, Prefix);
 
-                if (!model.Options.IsJson())
+                try
                 {
-                    context.Updater.ModelState.AddModelError(Prefix + "." + nameof(MonacoSettingsViewModel.Options), S["The options are written in an incorrect format."]);
+                    var options = model.Options.Trim();
+
+                    if (!options.StartsWith('{') || !options.EndsWith('}'))
+                    {
+                        throw new Exception();
+                    }
+
+                    var engine = new Engine()
+                        .Execute("var config = " + options + "; config.language = 'html';");
```

Comment:
Insert a new line before `config.language ...` in case the input ends with `//`

Reply:
@sebastienros I removed the Engine here since we are going to be parsing JSON without any support to a JS callback. so they user should be able to provide JSON.